frontend/src/main/java/com/google/tripmeout/frontend/TripModel.java

```diff
@@ -9,9 +9,9 @@ public static Builder builder() {
     return new AutoValue_TripModel.Builder();
   }
 
-  public abstract String id();
-  @Nullable public abstract String name();
-  public abstract String userId();
+  @Nullable public abstract String id();
+  public abstract String name();
+  @Nullable public abstract String userId();
```

Comment:
We want the user id to be nullable?

Reply:
I don't think we expect the client to provide the userId as part of the request payload itself.